0847_deducing_this/deducing-this.bs

```diff
@@ -273,13 +273,45 @@ void ex(X& x, D const& d) {
 
 Member functions with an explicit object parameter cannot be `static` or have *cv*- or *ref*-qualifiers. 
 
-A call to a member function will interpret the object parameter as the first argument to such a member function, the first argument in the parenthesized expression list as the second parameter, and so forth. 
+A call to a member function will interpret the object argument as the first (`this`-annotated) parameter to such a member function, the first argument in the parenthesized expression list as the second parameter, and so forth. 
 
 Following normal deduction rules, the template parameter corresponding to the explicit object parameter can deduce to a type derived from the class the member function is declared in (as in the example above for `d.bar()`).
 
+
+### Explicit object parameter, sans identifier ### {#explicit-object-parameter-no-id}
```

Comment:
I thought BSI hated this syntax, or am I making this up?

Reply:
They did, but Herb loved it. I figured including it is not a bad idea.

Reply:
Yeah, but I feel like the two trailing type syntaxes are at least viable solutions - even if we don't want them. This one I don't think is even viable (unless we go back in time and change what `this` means - in which case it would retroactively the best solution I think).

Reply:
Yes, Herb suggested exactly that - `this` is a reference and behaves exactly the way `self` does in the explicit parameter syntax. In lambdas it shadows captured `this`, since it's a parameter, so you'd need to refer to captured `this` using an assignment alias.

Reply:
Ok I'm just going to merge this for now and think about it.